### DISCUSSION TITLE
v2.1.4

### DISCUSSION THREAD

**Discussion by ruslandoga** [plausible/analytics#4663] [Self-Hosted Support]
This is a patch release focused on resolving file moving [issue](https://github.com/plausible/analytics/issues/4638) with CSV export/import, alongside several other improvements.

#### Upgrade

Update the image used for `plausible`

<sub><kbd>compose.yml</kbd></sub>
```diff
plausible:
- image: ghcr.io/plausible/community-edition:v2.1.3
+ image: ghcr.io/plausible/community-edition:v2.1.4
```

and recreate the containers

<sub><kbd>console</kbd></sub>
```console
$ cd plausible-ce # location of https://github.com/plausible/community-edition
$ docker compose up -d
```

This will boot up the new version of the app.

## Changelog

Following [changes](https://github.com/plausible/analytics/compare/v2.1.3...v2.1.4) have been made since v2.1.3:

### Added

- Add ability to review and revoke particular logged in user sessions
- Add ability to change password from user settings screen
- Add error logs for background jobs plausible/analytics#4657

### Changed

- Revised User Settings UI
- Default to `invite_only` for registration plausible/analytics#4616

### Fixed

- Fix cross-device file move in CSV exports/imports plausible/analytics#4640

<hr /><em>This discussion was created from the release <a href='https://github.com/plausible/analytics/releases/tag/v2.1.4'>v2.1.4</a>.</em>

**Comment by nsgoyat** [comment]
Has anyone noticed that the stats screen has an empty area outside the viewport on mobile screens? While using the Plausible v2.1.2+ versions on the smartphone, I found the display always scrolling horizontally.

**Reply by ruslandoga** [reply]
👋 @nsgoyat 

Would you be able to make a screenshot? Also, does https://plausible.io/plausible.io have that problem? FWIW I tried the latter just now from an iPhone and didn't see any extra empty areas.

**Reply by nsgoyat** [reply]
Hey @ruslandoga 
Sorry, I don't have the same issue with the URL you provided, but I have two different self-hosted instances. One runs the latest version, and one runs the V2.1.2, and both have the issue.

Here is a screen record of those panels: https://photos.app.goo.gl/3o4J3h713YrEvCDj6

**Reply by priard** [reply]
I can confirm that I'm experiencing the same issue across multiple mobile devices.

**Reply by nsgoyat** [reply]
@ruslandoga May you please try deploying a server with the latest version of the Plausible CE and see if the scrolling issue on mobile persists? I tried it with recent 2-3 versions and they all have this bug.

**Reply by ruslandoga** [reply]
👋 @nsgoyat 

I tried opening https://who.edify.space/dummy.site which runs CE version `v2.1.4` and haven't been able to exactly reproduce. There is a vertical scroll indicator (it probably shouldn't be there at all) but it's small. Maybe the issue is with the data that's shown on the dashboard, like the longs strings and such? I'll try adding pageviews with really long paths.

---

I added https://who.edify.space/dummy.site?filters=((is,page,(//path/to/your/extraordinarily/lengthy/resource/which/keeps/going/on/and/on/with/lots/of/subdirectories/and/query/parameters))) but still seems like no extra vertical space is added to the dashboard.

**Comment by priard** [comment]
I noticed that Plausible already has API version v2. Is the new API version also planned for Plausible CE?

**Reply by ruslandoga** [reply]
👋 @priard 

API v2 is still in beta so it hasn't been announced yet in CE.

**Comment by ruslandoga** [comment]
👋 

Another patch release (v2.1.5) is now available: https://github.com/plausible/analytics/discussions/5041
